LEGO1/lego/legoomni/include/skateboard.h

```diff
@@ -37,7 +38,10 @@ class SkateBoard : public IslePathActor {
 private:
 	// TODO: SkateBoard types
 	undefined m_unk0x160;
-	undefined m_unk0x161[0x7];
+	undefined m_unk0x161;
```

Comment:
Since the members `m_unk0x161` through `m_unk0x163` are apparently not used, you can completely remove them. The compiler will insert padding in this case so that `m_unk0x164` remains at `0x164`.